.github/buildomat/jobs/deploy.sh

```diff
@@ -147,7 +147,7 @@ pfexec ipadm create-addr -T static -a 192.168.1.199/24 igb0/sidehatch
 # address for upstream connectivity.
 #
 tar xf out/omicron-sled-agent.tar pkg/config-rss.toml
-sed -e '/\[gateway\]/,/\[request\]/ s/^.*address =.*$/address = "192.168.1.199"/' \
+sed -e 's/^# address =.*$/address = "192.168.1.199"/' \
```

Comment:
Now that `config-rss.toml` has been shrinking, it may actually make sense to just have totally a distinct copy for the lab?

FYI to @andrewjstone , I do think when we get to the point where wicket actually talks to RSS to "let the operator dynamically set a profile (including Nexus IP info, the subnet, etc)", it'll be a great time to revisit "how we pass this info into RSS". 

I *think* the gateway stuff won't be necessary at that point, since it's only needed for the OPTE hack?

Reply:
Totally!